### ISSUE TITLE
Logs which are located in afterEach hooks, are not sent to reportportal

### ISSUE THREAD

**Issue by grib9544** [reportportal/agent-js-mocha#50]
Hello, 
I found out some bug, it's related with issue [#44](https://github.com/reportportal/agent-js-mocha/issues/44).

The solution to resolve the bug is to remove the [line](https://github.com/reportportal/agent-js-mocha/blob/45596d32f4b607332fd85a7bda75cb2021f32962/lib/mochaReporter.js#L296). The problem is that we remove currentTest, and logs in afterEach hooks can't be sent to reportportal when currentTest value is null.

If you have contribution guide, I can create pull request to resolve this problem. 

Regards,

**Comment by AmsterGet** [comment]
Hello @grib9544!
You can fork this repo and create a PR to the `develop` branch.
Update the tests if needed.
Also please add a note describing your feature/fix to the `CHANGELOG.MD`.
Also add me (@AmsterGet) as a reviewer.
Thanks for your contribution!

**Comment by ostrovanka** [comment]
Hi, I wanted to give some update on this issue.

I debugged this a little bit and this is what I've got:
Reporter configuration:
`reporterOptions: {
    token: 'f',
    endpoint: 'http://localhost:8080/api/v1',
    project: 'PIT',
    launch: `PIT Run`,
    skippedIssue: false,
    reportHooks: true,
}`

Logs for beforeEach/afterEach methods are added to test itself, beforeEach/afterEach are present in report but are empty

Logs for beforeAll/afterAll methods are not added anywhere at all, although they are also present in report. 

While debugging `mochaReporter.sendLog()` method I see that `tempId` for hook is different from `tempId` of test itself, but logs are still added to test, not hook.

So to sum it up:
- adding logs for beforeEach/afterEach - works partially, because logs are added to the wrong place
- adding logs for beforeAll/afterAll - seems like this doesn't work at all

**Comment by n0rm4l-real** [comment]
Hello guys, any update on this?

**Comment by had0uk3n** [comment]
also want to up this topic. still need logs in beforeEach/afterEach

**Comment by NeginMn** [comment]
@AmsterGet Can you please add this fix to the new version that is being released?
